File: src/Philips.IBE.Service.WebAgent/Philips.IBE.Service.WebAgent.Server/Models/Contract.cs
namespace Philips.IBE.Service.WebAgent.Server.Models
{
    public class Contract
    {
        public Acknowledgement? Acknowledgement { get; set; } 
        public HighFidelity? HighFidelity { get; set; } 
        public List<int>? InputIDs { get; set; }
        public string Name { get; set; } = string.Empty;
        public int OutputID { get; set; }
    }
}

File: src/Philips.IBE.Service.WebAgent/philips.ibe.service.webagent.client/src/app/models/Contract.ts
import { Acknowledgement } from "./Acknowledgement";
import { CommunicationData } from "./CommunicationData";
import { HighFidelity } from "./HighFidelity";
import { Workflow } from "./Workflow";

export interface Contract {

  acknowledgement: Acknowledgement,
  highFidelity: HighFidelity,
  inputIDs:Array<number>,
  name: string,
  outputID: number,
  output?: CommunicationData;
  input?: CommunicationData[];
}

